docs/source/general.md

```diff
@@ -0,0 +1,160 @@
+
+- sampleId
+- na_matrix_raw
+- rna_matrix_filtered
+- hto_matrix_raw
+- hto_matrix_filtered
+- bam
+- bam_index
+- barcodes
+- nsample
+- celldata
+- vcf_mixed
+- vcf_donor
+
+### **scverse compatibility**
+
+To ensure scverse compatibility, the pipeline provides the option to generate anndata or mudata after demultiplexing specifeid by `params.generate_anndata` and `params.generate_mudata`. This object contains the scRNA-seq counts from `params.rna_matrix_filered` and stores the assignment of each demultiplexing method in the `assignment` column of `obs`. Additionlly, if `match_donor` is True, the pipeline also produces an AnnData object which contains the assignment of the best-matched method pair after donor matching.
+
+## **Pipeline output**
+
+Output directory of the pipeline is set by `$params.outdir`. By default, the pipeline is run on a single sample. In this case, all pipeline output will be saved in the folder `$projectDir/$params.outdir/$params.mode`. When running the pipeline on multiple samples, the pipeline output will be found in the folder `"$projectDir/$params.outdir/$sampleId/$params.mode`. To simplify this, we'll refer to this folder as `$pipeline_output_folder` from now on.
+
+The demultiplexing workflow saves its output in `$pipeline_output_folder/[gene/hash]_demulti`. The pipeline will also generate some TSV files to summarize the results in the folder `[gene/hash]_summary` under this directory.
+
+- `[method]_classification.csv`: classification of all trials for a given method
+  | Barcode | multiseq_1 | multiseq_2 | ... |
+  |:---------: |:----------: |:----------: |:---: |
+  | barcode-1 | singlet | singlet | ... |
+  | barcode-2 | doublet | negative | ... |
+  | ... | ... | ... | ... |
+- `[method]_assignment.csv`: assignment of all trials for a given method
+  | Barcode | multiseq_1 | multiseq_2 | ... |
+  |:---------: |:----------: |:----------: |:---: |
+  | barcode-1 | donor-1 | donor-2 | ... |
+  | barcode-2 | doublet | negative | ... |
+  | ... | ... | ... | ... |
+- `[method]_params.csv`: specified paramters of all trials for a given method
+  | Argument | Value |
+  | :---------: | :----------: |
+  | seuratObejctPath | Path |
+  | quantile | 0.7 |
+  | ... | ... |
+- `[workflow]_classification_all.csv`: classification of all trials across different methods
+  | Barcode | multiseq_1 | htodemux_1 | ... |
+  |:---------: |:----------: |:----------: |:---: |
+  | ... | ... | ... | ... |
+- `[workflow]_assignment_all.csv`: save the assignment of all trials across different methods
+  | Barcode | multiseq_1 | htodemux_1 | ... |
+  |:---------: |:----------: |:----------: |:---: |
+  | ... | ... | ... | ... |
+- `adata` folder: stores Anndata object with filtered scRNA-seq read counts and assignment of each deconvolution method if `params.generate_anndata` is `True`. Details see section "scverse compatibility" above.
```

Comment:
Does the equivalent not exist for mudata?

Reply:
still in Mariana' branch, not merged yet. But i can add in the doc now